gateway/crates/config/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+edition.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license = "MPL-2.0"
+name = "gateway-config"
+repository.workspace = true
+version = "0.6.1"
+
+[lints]
+workspace = true
+
+[dependencies]
+ascii = { version = "1.1.0", features = ["serde"] }
+duration-str = "0.11.0"
+engine.workspace = true
+grafbase-telemetry = { workspace = true, features = ["tower", "otlp"] }
+http.workspace = true
+parser-sdl = { version = "0.1.0", path = "../../../engine/crates/parser-sdl" }
```

Comment:
Should whatever needs this crate to actually get moved to parser-sdl? Would be great if the whole workspace depends on the config, not the other way around.

Reply:
Also workspace would be nicer here than path.

Reply:
did the workspace, but parser-sdl is kind of required anyway today. We're passing through SDL config to build the engine configuration today. And some tests for entity caching directly rely on stuff from `parser-sdl`. So well not fixing it in this PR